packages/common/src/hub-types.ts

```diff
@@ -360,22 +360,6 @@ export interface IHubTeam extends IGroup {
   };
 }
 
-/**
- * Defines a generic search response interface with parameterized result type
- * for different types of searches
- *
- * total - total number of results
- * results - potentially paginated list of results
- * hasNext - boolean flag for if there are any more pages ofresults
- * next - invokable function for obtaining next page of results
- */
-export interface ISearchResponse<T> {
```

Comment:
This interface is superseded by IHubSearchResponse<T> and was only used in an internal helper. Removing.